src/plugins/ui_feature/data_cleanup.ts

```diff
@@ -202,4 +206,43 @@ export class DataCleanupPlugin extends UIPlugin {
     }
     return CommandResult.Success;
   }
+
+  private trimWhitespace() {
+    const zones = this.getters.getSelectedZones();
+    const sheetId = this.getters.getActiveSheetId();
+    let count = 0;
+    for (const zone of zones) {
+      positions(zone).map(({ col, row }) => {
+        count += this.trimCell({ col, row, sheetId }) ? 1 : 0;
+      });
+    }
+    const text = count
+      ? _t("Trimmed whitespace from %s cells.", count)
+      : _t("No selected cells had whitespace trimmed.");
+    this.ui.notifyUI({
+      type: "info",
+      text: text.toString(),
+      sticky: false,
+    });
+  }
+
+  // Trims whitespace from cell content and returns true if the content has changed.
```

Comment:
only `/** */` comments are recognized as proper docstrings by IDEs. With `//` if you hover the function name, you won't see the docstring appear
```suggestion
  /**
   * Trims whitespace from cell content and returns true if the content has changed.
   */
```